Products/Zuul/tree.py

```diff
@@ -236,7 +236,7 @@ def __init__(self, context):
         self.model_catalog = IModelCatalogTool(self.context)
 
     def getBrain(self, path):
-        return self.model_catalog.getBrain(path)
+        return self.model_catalog.global_catalog.getBrain(path)
```

Comment:
that is not a legacy catalog. it is a helper:
https://github.com/zenoss/zenoss-prodbin/blob/develop/Products/Zuul/catalog/model_catalog_tool.py#L69-L71
https://github.com/zenoss/zenoss-prodbin/blob/develop/Products/Zuul/catalog/model_catalog_tool_helper.py#L14